README.md

```diff
@@ -0,0 +1,33 @@
+# Open Pension Scraper
+
+[![Build Status][travis-image]][travis-url]
+
+Just like [Open Pension Scraper](https://github.com/nirgn975/open_pension_scraper#readme),
```

Comment:
I don't think you need to mention the `Open Pension Scraper` project, this project stand on it's own.